src/app/api/auth/reset-password/route.ts

```diff
@@ -0,0 +1,55 @@
+import { NextResponse } from 'next/server'
+import { resetPassword } from '@/helpers/api/service/auth-service'
+
+/**
+ * @swagger
+ * /api/auth/reset-password:
+ *   post:
+ *      tags: [authorization]
+ *      summary: Reset password
+ *      description: Reset password.
+ *      requestBody:
+ *        content:
+ *          application/json:
+ *            schema:
+ *               email: string
+ *               newPassword: string
+ *            example:
+ *              email: test@test.com
+ *              newPassword: password123
+ *      responses:
+ *        '200':
+ *          description: OK
+ *          content:
+ *            application/json:
+ *             schema:
+ *               email: string
+ *               userId: string
+ *               userName: string
+ *             example:
+ *               email: test@test.com
+ *               userId: 123h
+ *               userName: user
+ */
+export async function POST(req: Request) {
+  try {
+    const result = await req.json()
+    const { email, newPassword } = result
+    const userData = await resetPassword(email, newPassword)
```

Comment:
read the comments above about reusability

Reply:
fixed